boards/Makefile.common

```diff
@@ -148,7 +148,7 @@ debug-lst: target/$(TARGET)/debug/$(PLATFORM).lst
 
 .PHONY: doc
 doc: | target
-	$(Q)RUSTDOCFLAGS=--document-private-items $(CARGO) doc $(VERBOSE) --release --target=$(TARGET)
```

Comment:
I'm not sure what was the meaning of this flag, but could you explained why you had to remove it?

Reply:
Ok, just found the explanation in the commit history.